doc/classes/MultiMesh.xml

```diff
@@ -109,6 +109,12 @@
 	</methods>
 	<members>
 		<member name="buffer" type="PackedFloat32Array" setter="set_buffer" getter="get_buffer" default="PackedFloat32Array()">
+			The raw buffer data for this multimesh object. 
+			
+			This property provides access to the underlying buffer that stores transformation 
+			matrices and other instance data for all mesh instances.
+
+			The buffer structure depends on the [member transform_format] property and if [member use_custom_data] or [member use_colors] is enabled. More info in [member RenderingServer.multimesh_set_buffer].
```

Comment:
Only use one line per paragraph, as this is how the class reference is formatted:

```suggestion
			The raw buffer data for this multimesh object. This property provides access to the underlying buffer that stores transformation matrices and other instance data for all mesh instances.
			The buffer structure depends on the [member transform_format] property and if [member use_custom_data] or [member use_colors] is enabled. See also [member RenderingServer.multimesh_set_buffer].
```